src/smart-app-banner/types.ts

```diff
@@ -0,0 +1,17 @@
+export interface SmartAppBannerProps {
+    show: boolean;
+    link: string;
+    content: SmartAppBannerContentProps;
+    offset?: number | undefined;
+    className?: string | undefined;
+    onBannerDismiss: () => void;
```

Comment:
actually a question, is this required to be mandatory?

Reply:
yes, because there is no default set. we need to pass a callback which sets `show` to false

Reply:
updated